x-pack/plugins/apm/server/routes/suggestions.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+import { maxSuggestions } from '../../../observability/common';
+import { getSuggestions } from '../lib/suggestions/get_suggestions';
+import { getSearchAggregatedTransactions } from '../lib/helpers/aggregated_transactions';
+import { setupRequest } from '../lib/helpers/setup_request';
+import { createApmServerRoute } from './create_apm_server_route';
+import { createApmServerRouteRepository } from './create_apm_server_route_repository';
+
+const suggestionsRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/suggestions',
```

Comment:
Thanks for creating this as an internal api.
I recall there was an issue for doing this in Kibana but can't find it now. I think we should have an APM specific issue so we have it captured on our board too.

Reply:
Created an issue for internal apis here: https://github.com/elastic/kibana/issues/113383